### PR TITLE
Distill.py with an explicit list of n-tuples

### PR DESCRIPTION
**Pull request by mblaszkiewicz**
Here is the other version of distill.py with opening n-tuples from the list in input_files.txt

### CODE REVIEW

File: distill.py [hunk 1]
```diff
@@ -3,98 +3,128 @@
...
-    "d45b_56t"  : (["track_rp_5",   "track_rp_21",  "track_rp_25"],
-                   ["track_rp_104", "track_rp_120", "track_rp_124"]),
-    "ad45b_56b" : (["track_rp_5",   "track_rp_21",  "track_rp_25"],
-                   ["track_rp_104", "track_rp_120", "track_rp_124"]),
-    "d45t_56b"  : (["track_rp_4",   "track_rp_20",  "track_rp_24"],
-                   ["track_rp_105", "track_rp_121", "track_rp_125"]),
-    "ad45t_56t" : (["track_rp_4",   "track_rp_20",  "track_rp_24"],
-                   ["track_rp_105", "track_rp_121", "track_rp_125"])
-}
+# Convert to PyROOT vector
```
Comment (JavierCVilla) [inline distill.py]: Although I think your approach is more convenient for an interactive execution in a Python Notebook,  for a single script it might be better to follow the current approach (using one dictionary with all diagonals and tracks). 

One reason is that we want to produce the same result as `distill.cc` where we run one single diagonal per execution. So I will add here a parser to accept arguments where we can reproduce the options from `distill.cc`.

In summary, could you please modify `distill.py` just to consider the new `source_file`?

**Review by JavierCVilla — CHANGES_REQUESTED** [review]
Hi @mblaszkiewicz 

Thanks a lot for this PR, can you please consider the requested change? Let me know if you see any cons with the current approach.

File: distill.py (hunk 1 above)
Comment (mblaszkiewicz) [reply distill.py]: Sure, the approach from the commit does not make sense in a script file at all, I thought you wanted it to be committed just for future reference. In a moment I will push the distill.py only with the new source_file

**Comment by mblaszkiewicz** [comment]
Sorry, commit above is not the one I wanted to push here (although with new input_files, it's based on master branch)

**Comment by JavierCVilla** [comment]
Looks fine to me @mblaszkiewicz. Thanks!
